duo-client/pom.xml

```diff
@@ -44,7 +44,7 @@
     <dependency>
       <groupId>com.squareup.okhttp3</groupId>
       <artifactId>okhttp</artifactId>
-      <version>4.9.1</version>
+      <version>4.10.0</version>
```

Comment:
https://security.snyk.io/vuln/SNYK-JAVA-COMSQUAREUPOKHTTP3-2958044
It recommend to upgrade to 4.9.2 or higher. So I just bump to latest release. I can upgrade to 4.9.2 if it is too risky to jump to the latest

Reply:
I think this is fine, https://security.snyk.io/package/maven/com.squareup.okhttp3:okhttp/4.10.0 says there are no vulnerabilities and it was released june of last year. They're already working on 5.0.0-alpha versions, so I think 4.10.0 is safe.